### PR TITLE
Fix blocksyncer cfg

### PR DESCRIPTION
**Pull request by constwz**
## Description

add a description of your changes here...

## Rationale

tell us why we need these changes...

### CODE REVIEW

File: docs/guide/storage-provider/run-book/config.md [hunk 1]
```diff
@@ -325,13 +307,11 @@ BsDBSwitchCheckIntervalSec = 0
 # required
 Modules = []
 # required
-Dsn = ''
-# optional
-DsnSwitched = ''
+DBAddress = 'localhost:3306'
```
Comment (sysvm) [inline docs/guide/storage-provider/run-book/config.md:310]: This address is code gen?

**Review by clydemeng — APPROVED** [review]

File: docs/guide/storage-provider/run-book/config.md (hunk 1 above)
Comment (constwz) [reply docs/guide/storage-provider/run-book/config.md]: fixed

**Review by BarryTong65 — APPROVED** [review]

**Review by KeefeL — APPROVED** [review]

File: docs/guide/storage-provider/run-book/config.md [hunk 2]
```diff
@@ -424,11 +403,11 @@ AuthenticatorEndpoint = 'localhost:9333'
...
 
 ## BlockSyncer
-Here is how you put `Dsn` config for `[BlockSyncer]`
+Here is block_syncer config.
+The configuration of DBAddress, DBName, and BSDB can be the same. To improve performance, you can configure the address of the write database here, and the address of the read database on the corresponding BSDB
 ```
 Modules = ['epoch','bucket','object','payment','group','permission','storage_provider','prefix_tree', 'virtual_group','sp_exit_events','object_id_map']
-Dsn = "user:passwd*@tcp(localhost:3306)/block_syncer?parseTime=true&multiStatements=true&loc=Local&interpolateParams=true"
-DsnSwitched = ''
+DBAddress = 'localhost:3306'
```
Comment (sysvm) [inline docs/guide/storage-provider/run-book/config.md:410]: Where is db password?

Comment (KeefeL) [reply docs/guide/storage-provider/run-book/config.md:410]: User and Password are needed?

File: docs/guide/storage-provider/run-book/config.md [hunk 3]
```diff
@@ -424,11 +403,11 @@ AuthenticatorEndpoint = 'localhost:9333'
 * `P2PBootstrap` can be left empty.
 
 ## BlockSyncer
-Here is how you put `Dsn` config for `[BlockSyncer]`
+Here is block_syncer config.
+The configuration of DBAddress, DBName, and BSDB can be the same. To improve performance, you can configure the address of the write database here, and the address of the read database on the corresponding BSDB
```
Comment (KeefeL) [inline docs/guide/storage-provider/run-book/config.md:407]: The configuration of DBAddress and DBName can be the same as the BSDB module here. To enhance performance, you can set up the write database address here and the corresponding read database address in BSDB.

**Review by sysvm — APPROVED** [review]
LGTM

**Pull request merged**